tui/tui.go

```diff
@@ -30,8 +31,7 @@ func New() (*TUI, error) {
 		return nil, err
 	}
 
-	os.Stdout = wout
-	os.Stderr = wout
+	// Redirect of stderr and stdout will happen in (t *TUI) Start(cmder ctl.Commander)
```

Comment:
@rdnt I moved this to `tui.Start()` so that logging still works before calling `start`

Reply:
Seems like this change was reverted